components/FlexibleHeader/src/MDCFlexibleHeaderView+ShiftBehavior.h

```diff
@@ -38,6 +38,16 @@ typedef NS_ENUM(NSInteger, MDCFlexibleHeaderShiftBehavior) {
    MDCFlexibleHeaderShiftBehaviorEnabled.
    */
   MDCFlexibleHeaderShiftBehaviorEnabledWithStatusBar,
+
+  /**
+   Allows the header to be shifted on- and off-screen only via the @c shiftHeaderOnScreenAnimated:
+   and @c shiftHeaderOffScreenAnimated APIs. Scroll events will not affect the visibility of the
+   header.
+
+   Analogous to UINavigationController's setNavigationBarHidden: behavior, in that the visibility of
+   the navigation bar persists regardless of the user's subsequent interactions.
+   */
+  MDCFlexibleHeaderShiftBehaviorHideable,
 };
```

Comment:
can we add a "Defaults to X" for the shiftBehavior property below?

Reply:
Added test to validate defaults in https://github.com/material-components/material-components-ios/pull/9036. Will send another PR to add the defaults docs.

Reply:
Default in https://github.com/material-components/material-components-ios/pull/9037